packages/examples/device_metrics/sensor_tag_ble_dell5k_mqtt_iotcc.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
```

Comment:
Just rename the path as "packages/examples/metrics".As you are making the new folder I`ll prefer if you can move the entities in the specific folder.